packages/visual-editor/src/vite-plugin/defaultLayoutData.ts

```diff
@@ -2331,7 +2331,7 @@ const mainDefaultLayout = {
                                   image: {
                                     field: "",
                                     constantValue: {
-                                      url: "https://images.unsplash.com/photo-1755745360285-0633c972b0fd?ixlib=rb-4.1.0&q=85&fm=jpg&crop=entropy&cs=srgb",
+                                      url: "https://images.unsplash.com/photo-1755745360285-0633c972b0fd?ixlib=rb-4.1.0&q=85&fm=jpg&crop=entropy&cs=srgb&w=1000",
```

Comment:
Similar comment that these widths are larger than what we need. I thought the image component handles getting the right size. I forget how Colton implemented.